build.fsx

```diff
@@ -209,15 +210,65 @@ Target.create "Push" (fun _ ->
 
 
 Target.create "SelfCheck" (fun _ ->
-    let srcDir = Path.Combine(rootDir.FullName, "src") |> DirectoryInfo
-
-    let consoleProj = Path.Combine(srcDir.FullName, "FSharpLint.Console", "FSharpLint.Console.fsproj") |> FileInfo
-    printfn "Checking %s..." consoleProj.FullName
-    exec "dotnet" (sprintf "run lint %s" consoleProj.FullName) consoleProj.Directory.FullName
-
-    let coreProj = Path.Combine(srcDir.FullName, "FSharpLint.Core", "FSharpLint.Core.fsproj") |> FileInfo
-    printfn "Checking %s..." coreProj.FullName
-    exec "dotnet" (sprintf "run lint %s" coreProj.FullName) consoleProj.Directory.FullName
+    let runLinter () =
+        let srcDir = Path.Combine(rootDir.FullName, "src") |> DirectoryInfo
+
+        let consoleProj = Path.Combine(srcDir.FullName, "FSharpLint.Console", "FSharpLint.Console.fsproj") |> FileInfo
+        printfn "Checking %s..." consoleProj.FullName
+        exec "dotnet" (sprintf "run lint %s" consoleProj.FullName) consoleProj.Directory.FullName
+
+        let coreProj = Path.Combine(srcDir.FullName, "FSharpLint.Core", "FSharpLint.Core.fsproj") |> FileInfo
+        printfn "Checking %s..." coreProj.FullName
+        exec "dotnet" (sprintf "run lint %s" coreProj.FullName) consoleProj.Directory.FullName
+    
+    printfn "Run FsharpLint with defualt rules."
```

Comment:
@Mersho remember what I told you about wrt English language: infinitive vs imperative vs gerund?

Reply:
@Mersho also: typo "defualt"